src/lib/taggingUtils.ts

```diff
@@ -10,6 +10,7 @@ const ANNOTATION_LABEL = 'custom';
 // all the values will be lowercased by the API
 export const tagKeys: { [key: string]: string } = {
```

Comment:
Why is custom outside of this? Answered myself: because the 3rd party dev is not supposed to filter for it. ==> Shouldn't flag also be outside of this for now?